build.sbt

```diff
@@ -347,19 +347,20 @@ lazy val scalalib =
       assembleScalaLibrary := {
         import org.eclipse.jgit.api._
 
-        val s      = streams.value
-        val trgDir = target.value / "scalaSources" / scalaVersion.value
+        val s         = streams.value
+        val trgDir    = target.value / "scalaSources" / scalaVersion.value
+        val scalaRepo = sys.env.get("SCALAREPO").getOrElse("https://github.com/scala/scala.git")
```

Comment:
We use `SCALANATIVE_` prefix for environment variables that influence our builds. That makes it a bit more explicit.